bokeh/settings.py

```diff
@@ -175,6 +175,12 @@ def sign_sessions(self, default=False):
         '''
         return self._get_bool("SIGN_SESSIONS", default)
 
+    def perform_document_validation(self, default=True):
+        ''' Set whether Bokeh should perform validation checks on properties
+        and documents.
+        '''
+        return self._get_bool("PERFORM_DOCUMENT_VALIDATION", default)
+
     # Server settings go here:
 
     def bokehjssrcdir(self):
```

Comment:
Needs an update here (not autogenerated):

https://raw.githubusercontent.com/bokeh/bokeh/master/sphinx/source/docs/dev_guide/env_vars.rst

personal taste, but I'd be OK with something shorter, e.g. `BOKEH_VALIDATE_DOC` but I'm not strongly opposed to this more verbose var name if you prefer it.